bootstrap/templates/kubernetes/apps/storage/openebs/app/helmrelease.yaml.j2

```
@@ -0,0 +1,36 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: openebs
+spec:
+  interval: 30m
+  chart:
+    spec:
+      chart: openebs
+      version: 3.3.0
+      sourceRef:
+        kind: HelmRepository
+        name: openebs
+        namespace: flux-system
+  maxHistory: 2
+  install:
+    remediation:
+      retries: 3
+  upgrade:
+    cleanupOnFail: true
+    remediation:
+      retries: 3
+  uninstall:
+    keepHistory: false
+  values:
+    localprovisioner:
+      hostpathClass:
+        enabled: true
+        isDefaultClass: false
+{% if bootstrap_distribution == 'k3s' %}
+        basePath: "{{ bootstrap_local_storage_path | default('/var/lib/rancher/k3s/local-hostpath', true) }}"
+{% else %}
+        basePath: "{{ bootstrap_local_storage_path | default('/var/openebs/local', true) }}"
+{% endif %}
```

```suggestion
    localprovisioner:
      hostpathClass:
        enabled: true
        name: openebs-hostpath
        isDefaultClass: false
        basePath: "{{ bootstrap_local_storage_path | default('/var/openebs/local', true) }}"
```

1. We can keep the `basePath` the same and then include a task in the k3s ansible nuke playbook to delete this directory.
2. I also added the storageclass name here just to be declarative on this.

---

Feel free to add number 1, or I can checkout the branch and add it. I suppose the `nuke` for k0s would also need it as well.

---

I made your changes.  Just need to add the cleanup to the nuke task.

---

I added simple enough logic around the nuke playbook that should work with k0s now.